Automattic/vip-go-mu-plugins
# update/bump_ep

## Description

Brings in:
* https://github.com/Automattic/ElasticPress/pull/104
* https://github.com/Automattic/ElasticPress/pull/105

## Changelog Description

### Plugin Updated: Enterprise Search

* Rename edgeNgram to edge_ngram in Elasticsearch index mappings to avoid a deprecation warning
* Rename the EP Facets widget


## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1) indexing no longer shows deprication warnings - `dev-env exec -- wp vip-search index --setup`
